docs/resources/compute_instance.md

```diff
@@ -283,6 +283,10 @@ The following arguments are supported:
 * `agency_name` - (Optional, String, ForceNew) Specifies the IAM agency name which is created on IAM to provide
     temporary credentials for ECS to access cloud services. Changing this creates a new instance.
 
+* `power_action` - (Optional, String) Specifies what action needs to be done for the instance.
```

Comment:
Specifies the power action to be done for ...

Reply:
The description has updated, and the note info `The power_action is a one-time action.` is better than `The power_action parameter specifies a one-time action.`?